core/web/external_initiators_controller.go

```diff
@@ -20,8 +20,11 @@ type ExternalInitiatorsController struct {
 // Create builds and saves a new service agreement record.
 func (eic *ExternalInitiatorsController) Create(c *gin.Context) {
 	eir := &models.ExternalInitiatorRequest{}
-	if !eic.App.GetStore().Config.Dev() {
-		jsonAPIError(c, http.StatusMethodNotAllowed, errors.New("External Initiators are currently under development and not yet usable outside of development mode"))
+	if !(eic.App.GetStore().Config.Dev() || eic.App.GetStore().Config.FeatureExternalInitiators()) {
```

Comment:
I feel like the contrapositive would be easier to read here.

Reply:
If you think of it as an 'unless' statement its easier.  The contrapositive would use a series of not's and 'ands' and its more to process in my opinion.  Anyone else?

Reply:
I do prefer `if` statements in the positive and am not a fan of `unless` in ruby. This line is taking me a while to grok. We want both to be false, correct? That does seem easier to understand:

```
if !(eic.App.GetStore().Config.Dev() || eic.App.GetStore().Config.FeatureExternalInitiators())

if eic.App.GetStore().Config.Dev() == False && eic.App.GetStore().Config.FeatureExternalInitiators() == False

if !eic.App.GetStore().Config.Dev() && !eic.App.GetStore().Config.FeatureExternalInitiators()
```